### PR TITLE
Router should accept "render" option

### PR DESCRIPTION
**Pull request by pafflique**
https://github.com/reactjs/react-router/blob/master/docs/API.md#renderprops

### CODE REVIEW

File: react-router/react-router.d.ts [hunk 1]
```diff
@@ -96,6 +96,7 @@ declare namespace ReactRouter {
         createElement?: (component: RouteComponent, props: Object) => any
         onError?: (error: any) => any
         onUpdate?: () => any
+        render?: (props: Object) => any
```
Comment (vvakame) [inline react-router/react-router.d.ts]: please use `any` instead of `Object`.

Comment (pafflique) [inline react-router/react-router.d.ts]: Why there is
createElement?: (component: RouteComponent, **props: Object**) => any
then?

Comment (vvakame) [inline react-router/react-router.d.ts]: `render?: (props: Object) => any` to `render?: (props: any) => any`.

**Comment by RyanCavanaugh** [comment]
Hello and thank you for your contribution!

Due to an excessively long queue of pull requests that have become stale, we are "declaring bankruptcy" and closing all PRs opened before May 1, 2016. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

Going forward, we are committing to review or merge all PRs on a regular basis so this bankruptcy will not occur again. We apologize for the incovenience and hope you will continue to contribute to DefinitelyTyped in the future.

Thanks
The TypeScript and DefinitelyTyped teams

**Pull request closed without merging**